libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -85,7 +85,6 @@
 static NSString * const kSFOAuthGrantTypeAuthorizationCode      = @"authorization_code";
 static NSString * const kSFOAuthResponseTypeActivatedClientCode = @"activated_client_code";
 static NSString * const kSFOAuthResponseClientSecret            = @"client_secret";
```

Comment:
There's still an old IP bypass flow that utilizes the client_secret value, with an activation code.  I'm 95% sure that whole flow can go away too, but removing this value should wait on that work.